third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs

```diff
@@ -177,7 +177,12 @@ impl<'a> FunctionGenerator<'a> {
         //   to avoid this.
         self.pinned = ctx.fun.get_pinned_temps(/*include_drop*/ true);
         self.temps = (0..ctx.fun.get_parameter_count())
-            .map(|temp| (temp, TempInfo::new(self.temp_to_local(ctx, None, temp))))
+            .map(|temp| {
```

Comment:
There seems to be useful discussion here about why we don't want to add to source maps here: https://github.com/aptos-labs/aptos-core/issues/12541#issuecomment-2053926508

We should provide some comments here as well explaining the newly added boolean and why it is used the way it is.